doc/src/conf.py

```diff
@@ -54,7 +58,11 @@
     penwidth='0.3')
 
 # The names of modules to mock and hence avoid import.
-autodoc_mock_imports = ['pyopencl']
+autodoc_mock_imports = ['pyopencl',
+                        'h5py',
+                        'mpi4py',
+                        'pycuda'
```

Comment:
pycuda?

Reply:
This is leftover from trying to get the backwards compatibility to work, will remove